Aquality.WinAppDriver/tests/Aquality.WinAppDriver.Tests/Applications/Locators/CalculatorWindow.cs

```diff
@@ -1,22 +1,42 @@
-﻿using OpenQA.Selenium;
+﻿using Aquality.WinAppDriver.Elements.Interfaces;
+using Aquality.WinAppDriver.Windows;
+using OpenQA.Selenium;
 using OpenQA.Selenium.Appium;
 
 namespace Aquality.WinAppDriver.Tests.Applications.Locators
 {
-    public static class CalculatorWindow
+    public class CalculatorWindow : Window
     {
         public static By WindowLocator => By.TagName("Window");
 
-        public static By OneButton => By.Name("1");
+        public static By OneButtonLocator => By.Name("1");
 
-        public static By TwoButton => By.Name("2");
+        public static By TwoButtonLocator => By.Name("2");
 
-        public static By ThreeButton => By.Name("3");
+        public static By ThreeButtonLocator => By.Name("3");
 
-        public static By PlusButton => By.Name("+");
+        public static By PlusButtonLocator => By.Name("+");
 
-        public static By EqualsButton => By.Name("=");
+        public static By EqualsButtonLocator => By.Name("=");
 
-        public static By ResultsLabel => MobileBy.AccessibilityId("48");
+        public static By ResultsLabelLocator => MobileBy.AccessibilityId("48");
+
+        public ITextBox TextBoxRightArgument => ElementFactory.GetTextBox(By.XPath("//*[@AutomationId='49']"), "Right Argument");
+
+        public IButton ButtonOne => ElementFactory.GetButton(OneButtonLocator, "1");
```

Comment:
please name with the type after the unique name: OneButton, ResultsLabel and so on

Reply:
done

Reply:
done